File: backend/components/metiers/threed/templates/helpers/mocks/templates_mock.js
// templates_mock.js – Mocks pour tests unitaires et intégration des templates Threed (JS)
// Inclut : scénarios spécifiques, edge cases, CI/CD, audit, accessibilité

module.exports = {
  mockRapportAudit: { date: '2025-06-03', result: 'OK' },
  mockEmailNotification: { details: 'Nouvelle scène threed créée' },
  mockAccessibiliteAudit: { date: '2025-06-03' },
  mockServiceExport: { date: '2025-06-03' }
};
// Utilisable pour tests e2e, migration massive, CI/CD

File: backend/components/metiers/threed/templates/helpers/mocks/templates_mock.py
"""
templates_mock.py – Mocks pour tests unitaires et intégration des templates Threed (Python)
Inclut : scénarios spécifiques, edge cases, CI/CD, audit, accessibilité
"""

import pytest


@pytest.fixture
def mock_rapport_audit():
    return {"date": "2025-06-03", "result": "OK"}


@pytest.fixture
def mock_email_notification():
    return {"details": "Nouvelle scène threed créée"}


@pytest.fixture
def mock_accessibilite_audit():
    return {"date": "2025-06-03"}


@pytest.fixture
def mock_service_export():
    return {"date": "2025-06-03"}


# Utilisable pour tests e2e, migration massive, CI/CD
